src/engine/cancun.md

```diff
@@ -96,14 +94,16 @@ Refer to the response for [`engine_newPayloadV2`](./shanghai.md#engine_newpayloa
 
 This method follows the same specification as [`engine_newPayloadV2`](./shanghai.md#engine_newpayloadv2) with the addition of the following:
 
-1. Given the expected array of blob versioned hashes client software **MUST** run its validation by taking the following steps:
+1. Client software **MUST** return `-32602: Invalid params` error unless all parameters and their fields are provided with non-`null` values.
+
+2. Client software **MUST** return `-38005: Unsupported fork` error if the `timestamp` of the payload is less than the Cancun activation timestamp. This rule replaces similar one defined for `engine_newPayloadV1`.
```

Comment:
I think we should do the same kind of reorder for `getPayloadV3`, i.e. move this error to the top

Reply:
It means that we validate timestamp before any other fields. Sounds good to me.